api/app/postgresql.js

```diff
@@ -1,23 +1,37 @@
 const items = require("../db/items.json");
 const { Pool } = require("pg");
+const pino = require('pino');
+const logger = pino({ name: 'kuma-backend-pg' });
+const dns = require('dns');
+dnsPromises = dns.promises;
```

Comment:
Is `dnsPromises` declared anyway?

```suggestion
const dnsPromises = dns.promises;
```